doc/source/reference/migration_guide_from_1.x_to_2.0.rst

```diff
@@ -0,0 +1,117 @@
+This is a guide for people moving from NetworkX 1.X to NetworkX 2.0
+
+As you know we have made some major changes to the methods in the Multi/Di/Graph classes.
+Methods that used to return lists are removed and the methods that returned iterators
+are renamed(removing the _iter suffix).
+For example we hade two methods to get nodes of a graph, ``G.nodes()`` and ``G.nodes_iter()``. ``G.nodes_iter()`` is renamed to ``G.nodes()``, hence 
```

Comment:
spelling:  hade -> have      but maybe simplify whole line to something like:
For example, G.nodes() now returns an iterator and G.nodes_iter() has been removed.